graphs/enwiki_echo_distribution_of_response_time.json

```diff
@@ -0,0 +1,233 @@
+{
+  "graph_version": "0.6.0",
+  "slug": "enwiki_echo_distribution_of_response_time",
+  "name": "Distribution of response time",
+  "shortName": "",
+  "desc": "",
+  "notes": "",
+  "root": {
+    "nodeType": "canvas",
+    "disabled": false,
+    "children": [
+      {
+        "nodeType": "axis",
+        "disabled": false,
+        "options": {
+          "tickFormat": "YY",
+          "labelSize": 10,
+          "labelColor": "#666666",
+          "stroke": {
+            "width": 2,
+            "color": "#CACACA"
+          },
+          "dimension": "x",
+          "orient": "bottom"
+        }
+      },
+      {
+        "nodeType": "axis",
+        "disabled": false,
+        "options": {
+          "tickFormat": "YY",
```

Comment:
'YY' seems to suggest date (year)-based, but this left axis should just be numbers. Not sure what that format should be.